samples/ClientSample/Program.cs

```diff
@@ -2,68 +2,101 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
+using System.IO.Pipelines;
 using System.Linq;
-using System.Threading.Tasks;
-using System.Net.WebSockets;
-using System.Threading;
+using System.Net.Http;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Logging;
 
 namespace ClientSample
 {
     public class Program
     {
-        public static void Main(string[] args)
-        {
-            RunWebSockets().GetAwaiter().GetResult();
-        }
+        public static void Main(string[] args) => MainAsync(args).Wait();
 
-        private static async Task RunWebSockets()
+        public static async Task MainAsync(string[] args)
         {
-            var ws = new ClientWebSocket();
-            await ws.ConnectAsync(new Uri("ws://localhost:5000/chat/ws"), CancellationToken.None);
+            var baseUrl = "http://localhost:5000/chat";
```

Comment:
Right now it just adds `/poll` and `/send` on the end. I expect we'll tidy this up, that's why this is extremely early prototype code :).